maybe-release.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
We might want to consider to move files supporting the release / staging to some other repository to avoid duplication. Not sure if it is a good moment to try / introduce this.